### DISCUSSION TITLE
Buenas tardes!!, Quien quisiera hacer equipo?

### DISCUSSION THREAD

**Discussion by Crosiux** [Launch-X-Latam/MisionFrontEnd#571] [Team Posts]
Si alguien esta interesado en hacer equipo por fa responder y lo creo por teams, para aprender juntos y hacer proyectos

**Comment by JoselinMiel** [comment]
Estoy interesado el correo es ***@***.***

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________
From: Andres Felipe Nieto Jimenez ***@***.***>
Sent: Thursday, November 17, 2022 2:13:56 PM
Cc: Subscribed ***@***.***>
Subject: [Launch-X-Latam/mision-backend] Buenas tardes!!, Quien quisiera hacer equipo? (#108)


Si alguien esta interesado en hacer equipo por fa responder y lo creo por teams, para aprender juntos y hacer proyectos

**Comment by jcda23** [comment]
Me apunto al equipo si aun esta disponible lx10901@innovaccion.mx tengo algo de practica en frontend

**Comment by Alejandro-Ari** [comment]
Acá un integrante mas si aún hay cupo
lx00580@innovaccion.mx

**Comment by JMGamero** [comment]
Aca otro integrante si hay lugar! lx12587@innovaccion.mx

**Comment by otrinidadv** [comment]
Hola, ¿Todavía hay cupo? les dejo mi correo: lx02654@innovaccion.mx

**Comment by JuanRodriguezDV** [comment]
lx08166@innovaccion.mx yo quiero
